src/c++/perf_analyzer/genai-perf/docs/tutorial.md

```diff
@@ -30,58 +30,47 @@ OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 - [Profile GPT2 running on Triton + TensorRT-LLM](#tensorrt-llm)
 - [Profile GPT2 running on Triton + vLLM](#triton-vllm)
-- [Profile GPT2 running on OpenAI API-Compatible Server](#openai)
+- [Profile GPT2 running on OpenAI Chat Completions API-Compatible Server](#openai-chat)
+- [Profile GPT2 running on OpenAI Completions API-Compatible Server](#openai-completions)
 
 ---
 
 ## Profile GPT2 running on Triton + TensorRT-LLM <a id="tensorrt-llm"></a>
 
-### Running GPT2 on Triton Inference Server using TensorRT-LLM
+### Run GPT2 on Triton Inference Server using TensorRT-LLM
 
 <details>
 <summary>See instructions</summary>
 
-1. Run Triton Inference Server with TensorRT-LLM backend container:
+Run Triton Inference Server with TensorRT-LLM backend container:
 
 ```bash
-export RELEASE="yy.mm" # e.g. export RELEASE="24.03"
+export RELEASE="yy.mm" # e.g. export RELEASE="24.05"
```

Comment:
Can we change all the tags to 24.06 since. that is the latest one?
```suggestion
export RELEASE="yy.mm" # e.g. export RELEASE="24.06"
```

Reply:
Sure, will do